sds_data_manager/stacks/processing_stack.py

```diff
@@ -0,0 +1,113 @@
+"""Data Storage Stack
+This is the module containing the general stack to be built for
+computation of different algorithms
+"""
+from pathlib import Path
+
+from aws_cdk import (
+    Environment,
+    Stack,
+)
+from aws_cdk import (
+    aws_ec2 as ec2,
+)
+from aws_cdk import (
+    aws_events as events,
+)
+from aws_cdk import (
+    aws_events_targets as event_targets,
+)
+from aws_cdk import (
+    aws_s3 as s3,
+)
```

Comment:
```suggestion
from aws_cdk import Environment Stack
from aws_cdk import aws_ec2 as ec2
from aws_cdk import aws_events as events
from aws_cdk import aws_events_targets as event_targets
from aws_cdk import aws_s3 as s3
```

Is this some automatic formatting from the IDE?